apps/explorer/test/support/factory.ex

```diff
@@ -776,6 +776,30 @@ defmodule Explorer.Factory do
     }
   end
 
+  def forward_transfer_same_block_factory do
+    address = insert(:address)
+
+    %ForwardTransfer{
+      block: build(:block),
+      block_number: build(:block).number,
+      to_address_hash: address.hash,
+      value: Enum.random(1..100_000),
+      index: 0
```

Comment:
Let's increment the `index` so it's closer to real data.

See here for example: https://github.com/HorizenLabs/eon-explorer/blob/bfee19a90578b6e4ab1b3028b9e9e383acc90407/apps/explorer/test/support/factory.ex#L1056-L1058

Reply:
In the other 3 factory methods too

Reply:
yes, agree. Index methods added